docs/references.bib

```diff
@@ -574,6 +574,24 @@ @Book{            coxlittleoshea1997
   isbn          = {978-0-387-94680-1}
 }
 
+@Article{         crans2017,
+  author        = {Crans, Alissa S. and Mukherjee, Sujoy and Przytycki,
+                  J\'{o}zef H.},
+  title         = {On homology of associative shelves},
+  journal       = {J. Homotopy Relat. Struct.},
+  fjournal      = {Journal of Homotopy and Related Structures},
+  volume        = {12},
+  year          = {2017},
+  number        = {3},
+  pages         = {741--763},
+  issn          = {2193-8407},
+  mrclass       = {18G60 (20M32 20N02 57M25)},
+  mrnumber      = {3691304},
+  mrreviewer    = {Mahender Singh},
+  doi           = {10.1007/s40062-016-0164-9},
+  url           = {https://doi-org.proxy.lib.ohio-state.edu/10.1007/s40062-016-0164-9}
```

Comment:
Rather than using the Ohio State DOI proxy, it'd be better to use a direct DOI link.
```suggestion
  url           = {https://doi.org/10.1007/s40062-016-0164-9}
```